packages/@lwc/integration-karma/README.md

```diff
@@ -40,6 +40,7 @@ This set of environment variables applies to the `start` and `test` commands:
 -   **`API_VERSION=<version>`:** API version to use when compiling.
 -   **`DISABLE_SYNTHETIC_SHADOW_SUPPORT_IN_COMPILER=1`:** Disable synthetic shadow in the compiler itself.
 -   **`DISABLE_STATIC_CONTENT_OPTIMIZATION=1`:** Disable static content optimization by setting `enableStaticContentOptimization` to `false`.
+-   **`ENABLE_EXPERIMENTAL_SIGNALS=1`:** Enables tests for experimental signals protocol.
```

Comment:
It just occurred to me: we do not technically need a new Karma env var, since `ENABLE_EXPERIMENTAL_SIGNALS` is a runtime-only flag. So you could just set the flag at runtime during the Karma tests.

Or do you intend to make it a compiler flag too in the future?

Reply:
TIL! No, I don't intend for there to be a compiler flag in the future.

I've removed it from the karma tests.